### DISCUSSION TITLE
What is contained in a signed snapshot?

### DISCUSSION THREAD

**Discussion by TrevorBenson** [input-output-hk/mithril#2192] [Q&A]
When the snapshot is issued and signed, does it contain only finalized blocks? E.g. on mainnet are the last 12 hours missing?

**Comment by jpraynaud** [comment]
Hi @TrevorBenson, it actually contains the full Cardano database at the time of the snapshot. 
When restored, the Cardano node makes sure that it runs on the main fork and retrieves the blocks produced since the snapshot was made.
